### PR TITLE
add github funding info

### PR DESCRIPTION
**Pull request by haoxusci**
- set github scilifelab kaw layout by buefy

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -134,19 +134,16 @@
...
-              src="/static/img/github.svg"
-              style="width: 36px; height:36px;margin-right:3px; margin-bottom:-8px;"
-            /><span style="font-size:1.3rem;">{{
-              siteConfig.footer_github.label
-            }}</span></a
+      <div class="columns is-moible" v-if="siteConfig.footer">
+        <div v-for="item in siteConfig.footer" :key="item.label" class="column">
+          <b-tooltip
+            :label="item.tooltip"
+            position="is-right"
```
Comment (oeway) [inline src/views/Home.vue:141]:
```suggestion
            position="is-top"
```

File: site.config.json [hunk 2]
```diff
@@ -16,10 +16,26 @@
...
+    {
+      "label": "Github",
+      "tooltip": "COVID-19 Model Repository",
+      "logo": "/static/img/github.svg",
+      "url": "https://github.com/CellProfiling/covid-models"
+    },
+    {
+      "label": "SciLifeLab",
+      "tooltip": "Science for Life Laboratory",
+      "logo": "/static/img/scilifelab.png",
```
Comment (oeway) [inline site.config.json:29]: Could you change it to a transparent version: https://www.pinclipart.com/pindetail/bxomhi_science-for-life-laboratory-logo-clipart/

File: src/views/Home.vue [hunk 3]
```diff
@@ -134,19 +134,16 @@
...
+            :href="item.url"
+            target="_blank"
           >
+            <img :src="item.logo" style="height: 55px;" />
```
Comment (oeway) [inline src/views/Home.vue:145]:
```suggestion
       <a  :href="item.url" target="_blank">
            <img :src="item.logo" style="height: 55px;" />
       </a>
```

File: src/views/Home.vue [hunk 4]
```diff
@@ -134,19 +134,16 @@
...
-            /><span style="font-size:1.3rem;">{{
-              siteConfig.footer_github.label
-            }}</span></a
+      <div class="columns is-moible" v-if="siteConfig.footer">
+        <div v-for="item in siteConfig.footer" :key="item.label" class="column">
+          <b-tooltip
+            :label="item.tooltip"
+            position="is-right"
+            :href="item.url"
+            target="_blank"
```
Comment (oeway) [inline src/views/Home.vue:143]: Remove these two, you need a `<a>` tag see below

**Comment by oeway** [comment]
We should perhaps also add a note to say the funding source explicitly `The COVID19AI.IO platform is kindly supported by the Science for Life Laboratory and the Knut and Alice Wallenberg Foundation.`

We can add another `footnote` key to the site config file.

File: src/views/Home.vue (hunk 1 above)
Comment (haoxusci) [reply src/views/Home.vue]: done

> It doesn't work well for the right-most item, use top instead:

File: site.config.json (hunk 2 above)
Comment (haoxusci) [reply site.config.json:29]: done

> Could you change it to a transparent version: https://www.pinclipart.com/pindetail/bxomhi_science-for-life-laboratory-logo-clipart/

File: src/views/Home.vue (hunk 3 above)
Comment (haoxusci) [reply src/views/Home.vue]: > 

done

**Comment by haoxusci** [comment]
> We should perhaps also add a note to say the funding source explicitly `The COVID19AI.IO platform is kindly supported by the Science for Life Laboratory and the Knut and Alice Wallenberg Foundation.`
> 
> We can add another `footnote` key to the site config file.

added

**Comment by oeway** [comment]
Could you also add the logo of our lab?

**Pull request closed without merging**